dev-python/pytest-flask/pytest-flask-1.3.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYPI_NO_NORMALIZE=1
+PYTHON_COMPAT=( python3_{11,12} pypy3 )
+inherit distutils-r1 pypi
+
+DESCRIPTION="A set of pytest fixtures to test Flask applications "
+HOMEPAGE="http://pytest-flask.readthedocs.org"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```

Comment:
That's implied by `distutils_enable_tests`. If you really insist on making it explicit, you need to add `RESTRICT="!test? ( test )"` as well.